libfwupdplugin/fu-path.h

```diff
@@ -48,6 +48,7 @@
  * @FU_PATH_KIND_HOSTFS_ROOT:		The root of the host filesystem (IE /)
  * @FU_PATH_KIND_HOSTFS_BOOT:		The host boot directory, (IE /boot)
  * @FU_PATH_KIND_DEVFS: 		The host dev directory, (IE /dev)
+ * @FU_PATH_KIND_LOCALTIME:		The timezone symlink (IE /etc/localtime)
```

Comment:
or `/var/lib/timezone/localtime` right?

Reply:
right, and that's only on ChromeOS -- you think it makes sense to add?

Reply:
I'd say it's better to have ChromeOS override `FU_PATH_KIND_LOCALTIME` in the environment then probably instead of hardcoding it

Reply:
Hmm, I'm hesitant to do that as it's not going to work OOTB. I'd be more willing to do it in the meson configure phase, but I don't know if /var/lib/timezone/localtime is going to exist on the build machine.

Reply:
got it, I guess the fallback is fine then.